internal/directory/google/google.go

```diff
@@ -49,14 +49,18 @@ func WithServiceAccount(serviceAccount *ServiceAccount) Option {
 
 // WithURL sets the provider url to use.
 func WithURL(url string) Option {
+	// if the empty string is sent in, use the default provider URL.
+	if url == "" {
+		url = defaultProviderURL
+	}
 	return func(cfg *config) {
 		cfg.url = url
 	}
 }
 
 func getConfig(opts ...Option) *config {
 	cfg := new(config)
-	WithURL(defaultProviderURL)(cfg)
+	WithURL("")(cfg)
```

Comment:
this would require me to peek to WithURL implementation to understand what is the magic behind empty argument.